JitsiConference.js

```diff
@@ -116,6 +117,19 @@ JitsiConference.prototype._init = function (options) {
     // Always add listeners because on reload we are executing leave and the
     // listeners are removed from statistics module.
     this.eventManager.setupStatisticsListeners();
+
+    if (this.options.config.enableTalkWhileMuted) {
+        this.talkMutedDetection
+            = new TalkMutedDetection(() => {
+                this.eventEmitter.emit(JitsiConferenceEvents.TALK_WHILE_MUTED);
+            });
+        this.statistics.addAudioLevelListener(
+            this.talkMutedDetection.audioLevelListener
+                .bind(this.talkMutedDetection));
```

Comment:
1. `JitsiConference`'s `statistics` doesn't appear to be (totally & absolutely) private. It's at least internal, package private (in the terms of Java) because `JitsiConferenceEventManager` uses it. I wouldn't stop you from accessing it inside `TalkMutedDetection` in order to add a listener. I realize though that it would add a dependency on top of accessing a property that is possibly intended to be private but... enlarging `JitsiConference` with each new feature...

2. However, why do you have to listen to `statistics` at all. Shouldn't you be able to get notified about audio level changes from the track itself?